tracer/test/Datadog.Trace.Tests/Debugger/LiveDebuggerTests.cs

```diff
@@ -51,18 +49,17 @@ public async Task DebuggerEnabled_ServicesCalled()
     [Fact]
     public async Task DebuggerDisabled_ServicesNotCalled()
     {
-        var tracerSettings = new TracerSettings(new NameValueConfigurationSource(new()
+        var settings = DebuggerSettings.FromSource(new NameValueConfigurationSource(new()
         {
-            { ConfigurationKeys.Debugger.Enabled, "0" },
+            { ConfigurationKeys.Debugger.Enabled, "1" },
```

Comment:
The test name says `Debugger Disabled` but now you are enabling the Live Debugger - Is it on purpose?

Reply:
Yare right, copy..paste issues.
Fixed here bf5ae3eef85e0ed47ae085a35662725ed36d0bd7